internal/cmd/backup_test.go

```diff
@@ -300,12 +300,13 @@ func TestBackupCreateCmdFunc(t *testing.T) {
 	_, err = c.WriteSchema(ctx, &v1.WriteSchemaRequest{Schema: testSchema})
 	require.NoError(t, err)
 
-	testRel := "test/resource:1#reader@test/user:1"
+	testRelString := "test/resource:1#reader@test/user:1"
+	testRel, _ := tuple.ParseV1Rel(testRelString)
```

Comment:
I'm gonna go back through these and use `tuple.MustParse().ToCoreTuple()` instead.

Reply:
Nvm it's a different type.